### ISSUE TITLE
About the velocity action controller

### ISSUE THREAD

**Issue by gaojunjie1999** [SJTU-ViSYS-team/VisFly#5]
Could you please help me with the following line:
![{1B79D5F1-98AF-421F-A127-337631F7BC04}](https://github.com/user-attachments/assets/f8734248-13e5-4906-a0a3-079ad53ec6d8)
The last term is the cross product of two identical vectors, which will turn out to be 0.
But shouldn't the last term be about 
![{756455B4-B49A-471C-9808-B189C404C7A6}](https://github.com/user-attachments/assets/3574768e-5388-48be-a48e-d0fd5e81ae6d)

**Comment by Fanxing-LI** [comment]
Yes, you are right. Sorry for this bug. Because I mainly use bodyrate or direct thrust control, I am unable to perform enough validation on this control method. I suggest to replace this line with:
```
body_torque_des = self._inertia @ (self._BODYRATE_PID.p @ pose_err + self._BODYRATE_PID.p @ ang_vel_err - cross(self._angular_velocity, self._inertia @ self._angular_velocity))
```
in both line 271 and line 243. You can try this line before i fix. $$\Omega \times J\Omega$$ is Coriolis effect, which is usually too minor to affect the simulation (it actually has impact but slight impact).
Because I have something else (exams) to do, today or tomorrow I cannot update it. I hope I could be free in this week.

Besides, #1 https://github.com/SJTU-ViSYS-team/VisFly/issues/1#issuecomment-2396958248, you are right. The flipud() should be deleted. I will update it in this week.
